docs/pom.xml

```diff
@@ -6,7 +6,7 @@
 	<parent>
 		<groupId>org.springframework.cloud</groupId>
 		<artifactId>spring-cloud-config</artifactId>
-		<version>1.3.3.BUILD-SNAPSHOT</version>
+		<version>1.4.0.BUILD-SNAPSHOT</version>
```

Comment:
While this change is unrelated, the build would not pass for me without making this change since it referenced the wrong parent version.